cmd/influx_inspect/help/help.go

```diff
@@ -38,6 +38,7 @@ The commands are:
     buildtsi             generates tsi1 indexes from tsm1 data
     help                 display this help message
     report               displays a shard level cardinality report
+    cardinality          estimates cloud 2 cardinality for a database
```

Comment:
I had expected just some extra flags on `report` - one for whole-database, and one for the cloud2-style counts

Reply:
Open to reasons why a new command makes more sense though.

Reply:
The separate command seemed cleaner to me; just an aesthetic and simplicity decision.  Rereading my code, though, I see I may be able to clean up the directory traversal code a little, but I still prefer a second command.

Reply:
`report` and `cardinality` as two different tools that report cardinality is not ideal.